.travis.yml

```diff
@@ -15,3 +15,13 @@ before_script:
 
 script:
   - cd queue-worker && ./build.sh
+
+after_success:
+    - if [ -z $DOCKER_NS ] ; then
```

Comment:
I'm not sure if this approach will work as expected (I've not tried it)

Is it possible to put the upper if statement within the context of the `if [ ! -z "$TRAVIS_TAG" ] ; then` statement?